pipelines/h6c/idr2/v3/post-lstbin/task_scripts/do_SINGLE_BASELINE_POSTPROCESSING_AND_PSPEC.sh

```diff
@@ -12,88 +12,31 @@ source ${src_dir}/_common.sh
-export OUT_PSPEC_FILE=${full_file_path%.uvh5}.pspec.h5
-export OUT_TAVG_PSPEC_FILE=${full_file_path%.uvh5}.tavg_pspec.h5
-export BAND_STR=${band_str}
-export ALREADY_INPAINTED=${already_inpainted}
-export PERFORM_INPAINT=${perform_inpaint}
-export INPAINT_MIN_DLY=${inpaint_min_dly}
-export INPAINT_HORIZON=${inpaint_horizon}
-export INPAINT_STANDOFF=${inpaint_standoff}
-export INPAINT_EIGENVAL_CUTOFF=${inpaint_eigenval_cutoff}
-export PERFORM_DLY_FILT=${perform_dly_filt}
-export DLY_FILT_MIN_DLY=${dly_filt_min_dly}
-export DLY_FILT_HORIZON=${dly_filt_horizon}
-export DLY_FILT_STANDOFF=${dly_filt_standoff}
-export DLY_FILT_EIGENVAL_CUTOFF=${dly_filt_eigenval_cutoff}
-export USE_BAND_AVG_NSAMPLES=${use_band_avg_nsamples}
-export FM_CUT_FREQ=${fm_cut_freq}
-export PIXEL_FLAG_CUT=${pixel_flag_cut}
-export CHANNEL_FLAG_CUT=${channel_flag_cut}
-export INTEGRATION_FLAG_CUT=${integration_flag_cut}
-export NINTERLEAVE=${ninterleave}
-export XTALK_FR=${xtalk_fr}
-export FR_SPECTRA_FILE=${fr_spectra_file}
-export FR_QUANTILE_LOW=${fr_quantile_low}
-export FR_QUANTILE_HIGH=${fr_quantile_high}
-export FR_EIGENVAL_CUTOFF=${fr_eigenval_cutoff}
-export TARGET_AVERAGING_TIME=${target_averaging_time}
-export USE_CORR_MATRIX=${use_corr_matrix}
-export CORR_MATRIX_FREQ_DECIMATION=${corr_matrix_freq_decimation}
-export CORR_MATRIX_NOTCH_CUTOFF=${corr_matrix_notch_cutoff}
-export EFIELD_HEALPIX_BEAM_FILE=${efield_healpix_beam_file}
-export TAPER=${taper}
-export INCLUDE_INTERLEAVE_AUTO_PS=${include_interleave_auto_ps}
-export STORE_WINDOW_FUNCTIONS=${store_window_functions}
-# TODO: add LST_RANGE_HOURS when we want to parameterize that
+
 
 # check if file is not just autocorrelaitons and that neither polarization is fully flagged
 if python ${src_dir}/check_single_bl_file.py ${full_file_path} --skip_autos --skip_outriggers; then
     # Execute jupyter notebook
-    nb_outfile=${full_file_path%.uvh5}.single_baseline_postprocessing_and_pspec.html
-    jupyter nbconvert --output=${nb_outfile} \
-    --to html \
-    --ExecutePreprocessor.allow_errors=False \
-    --ExecutePreprocessor.timeout=-1 \
-    --execute ${nb_template_dir}/single_baseline_postprocessing_and_pspec.ipynb
-    echo Finished running single baseline postprocessing and power spectrum estimation notebook for ${fn} at $(date) and writing results to ${nb_outfile}
+    nb_outfile=${full_file_path%.uvh5}.single_baseline_postprocessing_and_pspec
+    
+    runopts="--output-dir ${outdir} -k ${kernel} --toml ${toml_file} --toml-section ${toml_section}"
+    cfg="--basename ${nb_outfile} --SINGLE-BL-FILE ${full_file_path} --OUT-PSPEC-FILE=${full_file_path%.uvh5}.pspec.h5"
+    echo "Runopts: ${runopts}"
+    echo "Config: ${cfg}"
+
+    cmd="hnote run ${runopts} single_baseline_postprocessing_and_pspec ${cfg}"
+    echo $cmd
+    eval $cmd
+
+    echo Finished running single baseline postprocessing and power spectrum estimation notebook for ${fn} at $(date) and writing results to ${nb_outfile}.ipynb
```

Comment:
I'd suggest also writing the html path that was generated, as well as the .ipynb.